pkg/apis/build/v1alpha2/build.go

```diff
@@ -96,6 +96,17 @@ func (b *Build) BuiltImage() string {
 	return b.Status.LatestImage
 }
 
+func (b *Build) CacheImage() string {
+	if b == nil {
+		return ""
+	}
+	if !b.IsSuccess() {
+		return ""
```

Comment:
This is interesting. If the last build failed no cache image will be provided. This means the cache can get wiped with a single failed build due to a transient network error, etc. I wonder if we could find a way to make the provided cache image more resilient to build failures. This could be done by storing the last successful cache image or perhaps just providing the tag of cache image?

Reply:
We don't quite get it. The cache image will still be on the registry. It won't be updated with the failing build, but this is probably what we want, right?
This is used in `Build.Spec.LastBuild` which is used for `rebase`. But do we really need at a `Build.Spec.LastBuild.Cache` anyway? And also `Build.Spec` is immutable. So I think, we will just remove `Build.Spec.LastBuild.Cache`.